common/types.go

```diff
@@ -450,7 +450,7 @@ func (i *Big) Scan(value interface{}) error {
 }
 
 func (i Big) Value() (driver.Value, error) {
-	return i.Raw().Bytes(), nil
+	return i.Raw().GobEncode()
```

Comment:
@annielz Here we used GobEncode() method instead of Bytes() for negative value scenario. I think you need to decode big.int type property with GobDecode() method